instruction/simulatingTraffic/simulatingTrafficExample/generatePizzaTraffic.sh

```diff
@@ -60,6 +60,26 @@ while true; do
 done &
 pid4=$!
 
+# Simulate a failed pizza order every minute
+while true; do
+  response=$(curl -s -X PUT $host/api/auth -d '{"email":"d@jwt.com", "password":"diner"}' -H 'Content-Type: application/json')
+  token=$(echo $response | jq -r '.token')
+  echo "Login hungry diner..."
+
+  items='{ "menuId": 1, "description": "Veggie", "price": 0.05 }'
+  for (( i=0; i < 21; i++ ))
+  do items+=', { "menuId": 1, "description": "Veggie", "price": 0.05 }'
+  done
+  
+  curl -s -X POST $host/api/order -H 'Content-Type: application/json' -d "{\"franchiseId\": 1, \"storeId\":1, \"items\":[$items]}"  -H "Authorization: Bearer $token"
```

Comment:
Because you don't redirect the output it actually prints out the response. Maybe that isn't a problem and we should do this on the other ones also.

```sh
Login hungry diner...
{"message":"Failed to fulfill order at factory"}Bought too many pizzas...
Logging out hungry diner...
```